ophyd/ophydobj.py

```diff
@@ -588,14 +588,14 @@ def _repr_info(self):
         if self._parent is not None:
             yield ("parent", self.parent.name)
 
-    def __copy__(self):
```

Comment:
I said in the comment above: 

- I removed a weird __copy__ dunder
    - with the original __copy__ one test was not passing, because it recreates the object with different constructor arguments
    - I could not see why it was like this, no specific test neither
    
But, with my newest code the failing test works again. So I re-enabled `__copy__`, no prob.

(however, I still don't understand what it is for)